docs/src/modules/components/PickersPlayground.tsx

```diff
@@ -126,18 +125,20 @@ function BooleanGroupControl({
     [onChange],
   );
   return (
-    <FormControl
-      size="small"
-      sx={{
-        display: 'flex',
-        flexDirection: 'row',
-        alignItems: 'center',
-        justifyContent: 'space-between',
-        gap: 4,
-      }}
-    >
-      <FormLabel id={id}>{label}</FormLabel>
-      <Switch checked={value} onChange={handleChange} />
+    <FormControl>
+      <FormControlLabel
+        sx={{
+          justifyContent: 'space-between',
+          ml: 0,
+          width: '100%',
+        }}
+        label={label}
```

Comment:
Avoid usage of `FormLabel` as it produces grey label that has worse contrast, especially on light theme.